terraform/install.sh

```diff
@@ -174,6 +174,24 @@ applyTerraform() {
   terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"
 }
 
+installMonitoring() {
+  external_ip="";
+  while [ -z $external_ip ]; do
```

Comment:
I think you should use a `TRIES` variable and exit with a useful error message if it is never set up. I don't want the script to get stuck forever

Reply:
good idea - I've added a TRIES variable to try 20 times, retrieving the IP every 5 seconds so it'll spend at most 100 seconds on this block